include/cutlass/gemm/collective/xe_mma_w8a8.hpp

```diff
@@ -278,15 +272,14 @@ struct CollectiveMma<MainloopIntelW8A8<Stages, Schedule>, TileShape_, ElementA_,
       copy(mainloop.tiled_copy_a, tAgA(_,_,_,k_tile), tArA);
       copy(mainloop.tiled_copy_b, tBgB(_,_,_,k_tile), tBrB);
 
-      // TODO: register pressure
-      convert_FP8_to_FP16(tCrA, tCrA_fp16);
-      convert_FP8_to_FP16(tCrB, tCrB_fp16);
-
       if (prefetch_k < k_tile_count) {
         prefetch(tiled_prefetch_a, pAgA(_, _, _, prefetch_k));
         prefetch(tiled_prefetch_b, pBgB(_, _, _, prefetch_k));
       }
 
+      convert_FP8_to_FP16(tCrA, tCrA_fp16);
+      convert_FP8_to_FP16(tCrB, tCrB_fp16);
+

```

Comment:
Moving FP8 -> FP16 conversion after the prefetch block caused a huge regression. If this specific change is rolled back, performance improves ~21% on the default set of input shapes benchmarked by the example, but even after that, there's a ~6% regression compared to the commit before this one.